test/selenium_start.sh

```diff
@@ -28,7 +28,15 @@ systemctl start docker
 
 docker run -d -p 4444:4444 --name selenium-hub selenium/hub:3
 wait_curl /grid/console "Grid Console"
-docker run -d --link selenium-hub:hub --shm-size=512M selenium/node-chrome:3
-wait_curl /grid/console "browserName: chrome"
-docker run -d --link selenium-hub:hub --shm-size=512M selenium/node-firefox:3
-wait_curl /grid/console "browserName: firefox"
\ No newline at end of file
+
+if [ -z ${DEBUG_TEST+x} ]; then
```

Comment:
Note that this is a rather unusual syntax. `[ -z "$DEBUG_TEST" ]` seems easier to read and it's a much more common quoting style.